azkaban-common/src/main/java/azkaban/Constants.java

```diff
@@ -116,6 +116,17 @@
     public static final String AZKABAN_KEYTAB_PATH = "azkaban.keytab.path";
     public static final String PROJECT_TEMP_DIR = "project.temp.dir";
 
+    // Event reporting properties
```

Comment:
Could you add more detailed comments to this sections to explain how to use these configs? This can be a separate PR.